recipes-graphics/wayland/weston-init/weston.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Weston Wayland Compositor
+RequiresMountsFor=/run
+
+[Service]
+User=root
+EnvironmentFile=-/etc/default/weston
+ExecStartPre=/usr/bin/weston-waitforcard.sh
```

Comment:
Thanks for the response. What exactly means "didn't work out". Is there a bug in systemd, the documentation is not up to date or there may there was an issue how it was implemented?
The reason for having  systemd at all is to not to do this type of scripting.